draft-ietf-jose-hpke-encrypt.md

```diff
@@ -270,9 +276,9 @@ In HPKE JWE Key Encryption:
 - The "iv", "tag" JWE members MUST be present.
 - The "aad" JWE member MAY be present.
 - The "ciphertext" MUST be the base64url encoded ciphertext as described in Step 19 in Section 5.1 of {{RFC7516}}.
-- The HPKE Setup info parameter MAY be used, and its values are not constrained by this specification. By default, it is empty unless apu or apv are present, in which case it will carry the JOSE context-specific data as defined in Section 4.6.2 of {{RFC7518}}, i.e., the concatenation of AlgorithmID, PartyUInfo, and PartyVInfo. It does not include Z, keydatalen, SuppPubInfo, or SuppPrivInfo. AlgorithmID is structured as defined in Section 4.6.2 of {{RFC7518}} and the Data field in AlgorithmID will be set to the octets of the ASCII representation of the "alg" (algorithm) Header Parameter value.
+- The HPKE Setup info parameter MUST be set to an empty string.
```

Comment:
```suggestion
- The HPKE Setup info parameter SHOULD be set to an empty string. See RFC9180 Section 8.1 for details on how to provide auxiliary authenticated application information.
```

Reply:
The COSE-HPKE draft states that the setup info is set to an empty string. We should ensure that both drafts are aligned. In the last meeting, Mike was of the opinion that a new JOSE parameter would need to be added if auxiliary authenticated application information is required to be provided to HPKE setup info.

Reply:
I agree with making this consistent with COSE HPKE by making the parameter empty.  Let's deal with possibly relaxing this in a separate issue, if needed, but not in this PR.